contrib/endpoints/src/api_manager/config.cc

```diff
@@ -113,6 +113,36 @@ MethodInfoImpl *Config::GetOrCreateMethodInfoImpl(const string &name,
   return i->second.get();
 }
 
+bool Config::LoadQuotaRule(ApiManagerEnvInterface *env) {
```

Comment:
Thanks for your info, and let me make it clear: if there is rules, we do not care about anything in metric_rules, right? And we are always saving the quota info into a <metric, cost> pair, what should we use for the metric names for rules & metric_rules? It seems they have different structures.